content/platforms/kubernetes/cluster-recovery.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: Cluster Recovery in Kubernetes
+description: 
+weight: 40
+alwaysopen: false
+categories: ["Platforms"]
+aliases: /rs/concepts/kubernetes/cluster-recovery
+---
+
+## Cluster Recovery
+If the cluster losses a majority of its nodes (e.g. from nodes crashing or network split), the cluster will not responde. Recovering the cluster requires performing the following steps:
+
+1. Edit the rec resource, and set the clusterRecovery flag to true, for example by running
+
+> kubectl patch rec <cluster-name> --type merge --patch '{"spec":{"clusterRecovery":true}}'
+
+2. Wait for the cluster to recover, i.e. wait for all the pods to reach a Running status. Tip, to see the progress of the cluster run:
+
+> watch kubectl get pod
+
+3. Recover the data by running the following command from within any cluster pod
```

Comment:
we need to add the full flow kubectl exec and then rladmin

Reply:
@danni-m Something like: `kubeclt exec -it <pod-name>` "raldmin recover all"

Reply:
@otnielvh you can use bash and then rladmin, maybe its better if you plan to have more than 1 step with rladmin.
note that if you use a one liner you should use the full path for rladmin otherwise it will not work.

Reply:
Update the command according to Danni's comment?